src/lib/one-graph/netlify-graph.js

```diff
@@ -0,0 +1,705 @@
+
+  const variableSignature = typeScriptSignatureForOperationVariables(variableNames, fullSchema, operation)
+
+  const operationName = operation.name && operation.name.value
+
+  if (!operationName) {
+    throw new Error(`Operation name is required in ${basicFn.definition}\n\tfound: ${JSON.stringify(operation.name)}`)
+  }
+
+  const fn = {
+    ...basicFn,
+    fnName: makeFunctionName(operation.operation, operationName),
+    safeBody,
+    kind: operation.operation,
+    variableSignature,
+    returnSignature,
+    operationName,
+    parsedOperation: operation,
+  }
+
+  return fn
+}
+
+const generateJavaScriptClient = (netligraphConfig, schema, operationsDoc, enabledFunctions) => {
+  const operationsWithoutTemplateDollar = replaceAll(operationsDoc, '${', '\\${')
+  const safeOperationsDoc = replaceAll(operationsWithoutTemplateDollar, '`', '\\`')
+  const functionDecls = enabledFunctions.map((fn) => {
+    if (fn.kind === 'subscription') {
+      const fragments = []
+      return generateSubscriptionFunction(schema, fn, fragments)
+    }
+
+    const dynamicFunction = `${exp(netligraphConfig, fn.fnName)} = (
+  variables,
+  accessToken,
+  ) => {
+  return fetchOneGraph({
+    query: \`${fn.safeBody}\`,
+    variables: variables,
+    accessToken: accessToken
+  })
+}
+
+  `
+
+    const staticFunction = `${exp(netligraphConfig, fn.fnName)} = (
+  variables,
+  accessToken,
+) => {
+  // return fetchOneGraphPersisted(accessToken, "${fn.id}", "${fn.operationName}", variables)
+  return fetchOneGraph(accessToken, operationsDoc, "${fn.operationName}", variables)
+}
+
+`
+    return fn.id ? staticFunction : dynamicFunction
+  })
+
+  const exportedFunctionsObjectProperties = enabledFunctions
+    .map((fn) => {
+      const isSubscription = fn.kind === 'subscription'
+
+      if (isSubscription) {
+        const subscriptionFnName = subscriptionFunctionName(fn)
+        const parserFnName = subscriptionParserName(fn)
+
+        const jsDoc = replaceAll(fn.description || '', '*/', '')
+          .split('\n')
+          .join('\n* ')
+
+        return `/**
+  * ${jsDoc}
+  */
+  ${subscriptionFnName}:${subscriptionFnName},
+  /**
+   * Verify the event body is signed securely, and then parse the result.
+   */
+  ${parserFnName}: ${parserFnName}`
+      }
+
+      const jsDoc = replaceAll(fn.description || '', '*/', '')
+        .split('\n')
+        .join('\n* ')
+
+      return `/**
+  * ${jsDoc}
+  */
+  ${fn.fnName}: ${netligraphConfig.moduleType === 'commonjs' ? 'exports.' : ''}${fn.fnName}`
+    })
+    .join(',\n  ')
+
+  const dummyHandler = `${exp(netligraphConfig, 'handler')} = async (event, context) => {
+  // return a 401 json response
+  return {
+    statusCode: 401,
+    body: JSON.stringify({
+      message: 'Unauthorized',
+    }),
+  }
+}`
+
+  const source = `// GENERATED VIA \`netlify-plugin-netligraph\`, EDIT WITH CAUTION!
```

Comment:
No longer true?

Reply:
I'll change it to be, `GENERATED VIA netlify automated dev tools` if that reads better for you.